README.md

```diff
@@ -371,24 +383,24 @@ cloud_provider:
   name: aws
 ```
 
-AWS cloud provider has to be enabled on ec2 instances with the right IAM role.
+When using AWS cloud provider, all cluster nodes have to be assigned a proper IAM role.
 
 ### Azure Cloud provider
 
 Azure cloud provider can be enabled by passing `azure` as the cloud provider name and set of options to the configuration file:
 ```
 cloud_provider:
   name: azure
-  cloud_config:
-    aadClientId: xxxxxxxxxxxx
-    aadClientSecret: xxxxxxxxxxx
-    location: westus
-    resourceGroup: rke-rg
-    subnetName: rke-subnet
-    subscriptionId: xxxxxxxxxxx
-    vnetName: rke-vnet
-    tenantId: xxxxxxxxxx
-    securityGroupName: rke-nsg
+  azureCloudProvider:
```

Comment:
We need to put a clarification here of the change for the api across versions.

Reply:
As i said in #689, change **router** section to **route** in Openstack config example